app/src/bf-game/index.js

```diff
@@ -205,6 +244,20 @@ export function getCurrentPlayerId(gameState) {
   return gameState.players[gameState.turnCount % gameState.players.length];
 }
 
+export function getOpponentsAlive(gameState) {
+  const currentPlayerId = getCurrentPlayerId(gameState);
+  const opponentsAlive = [];
+  for (var playerId in gameState.statuses) {
+    if (
+      playerId === currentPlayerId ||
+      gameState.statuses[playerId] === PlayerStatus.Dead
+    ) {
+      continue;
+    } else opponentsAlive.push(playerId);
+  }
+  return opponentsAlive;
+}
+
```

Comment:
nitpick:
this could be done using a simple `filter` function 
```suggestion
export function getOpponentsAlive(gameState) {
  const currentPlayerId = getCurrentPlayerId(gameState);
  return gameState.players.filter(
    playerId =>
      playerId !== currentPlayerId &&
      gameState.statuses[playerId] !== PlayerStatus.Dead
  );
}

```